pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -132,11 +133,25 @@ func manageKubeSchedulerConfigMap_v311_00_to_latest(lister corev1listers.ConfigM
 	return resourceapply.ApplyConfigMap(client, recorder, requiredConfigMap)
 }
 
-func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, operatorConfig *operatorv1.KubeScheduler, imagePullSpec string) (*corev1.ConfigMap, bool, error) {
+func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, operatorConfig *operatorv1.KubeScheduler, imagePullSpec string, featureGateLister configlistersv1.FeatureGateLister) (*corev1.ConfigMap, bool, error) {
 	required := resourceread.ReadPodV1OrDie(v311_00_assets.MustAsset("v3.11.0/kube-scheduler/pod.yaml"))
 	if len(imagePullSpec) > 0 {
 		required.Spec.Containers[0].Image = imagePullSpec
 	}
+	// check for feature gates from feature lister.
+	featureGates, err := checkForFeatureGates(featureGateLister)
+	if err != nil || len(featureGates) == 0 {
+		// Default behaviour
+		klog.Infof("Error while getting feature gates or there are no featuregates for given feature, so defaulting to empty feature gates")
+		required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args, fmt.Sprintf("--feature-gates=%v", ""))
+	} else {
+		allFeatureGates := ""
```

Comment:
I have added this recently.

Reply:
featuregates need a true or false to designate whether they are enabled or disabled.

Reply:
Sorry, can you please elaborate? I am constructing a string with "," separated values here so that I can pass this onto `--feature-gates` flag to kube-scheduler.

Reply:
sorry. I'll comment in the right spot.